eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -162,6 +162,8 @@ class Peer(val nodeParams: NodeParams, remoteNodeId: PublicKey, watcher: ActorRe
 
       case Event(ChannelIdAssigned(channel, _, temporaryChannelId, channelId), d: ConnectedData) if d.channels.contains(TemporaryChannelId(temporaryChannelId)) =>
         log.info(s"channel id switch: previousId=$temporaryChannelId nextId=$channelId")
+        // we have a new channel with that peer: let's sync our routing table
+        d.peerConnection ! PeerConnection.DoSync
```

Comment:
I'm currently doing this at every channel creation. If we think it's too much, I can restrict that to only the first `FinalChannelId` (ie the first channel we open to that peer). Let me know what you prefer.

Reply:
I think we should only do it for the first channel, otherwise we'd be mostly wasting bandwidth, I'll change that.